Ticketing/Ticketing.UI/Controllers/CartsController.cs

```diff
@@ -87,8 +92,14 @@ public async Task<IActionResult> PutAsync(Guid cartId)
         [HttpDelete("{cartId}/events/{eventId}/seats/{seatId}")]
         public async Task<IActionResult> DeleteAsync(Guid cartId, int eventId, int seatId)
         {
-            await _cartService.DeleteSeatForCartAsync(cartId, eventId, seatId);
-            return Ok();
+            var result = await _cartService.DeleteSeatForCartAsync(cartId, eventId, seatId);
+
+            if (!result)
```

Comment:
tenar operator maybe? Also status code is enough - you do not need to send true/false

Reply:
done